instance/models/mixins/secret_keys.py

```diff
@@ -216,7 +216,7 @@ def http_auth_pass(self):
 
     def http_auth_info_base64(self):
         """
-        Return the HTTP auth information in the format required by Authorization HTTP header.
+        Return the HTTP auth information in the format required by Authorization HTTP header, as a string.
         """
         user_pass = '{}:{}'.format(self.http_auth_user, self.http_auth_pass)
-        return b64encode(user_pass.encode('latin1'))
+        return b64encode(user_pass.encode('latin1')).decode("utf-8")
```

Comment:
What we're encoding is user/pass which will be ASCII anyway, this is more about `str` to `bytes`.
Ok, I changed it to `utf-8` since if we're ever going to accept any non-ASCII character like `á` we'd also want to accept others like `ĉ` (non-latin1).